atlas/AtlasMl/tests/test_pipelines.py

```
@@ -229,16 +243,15 @@
             for title in titles
         ]
         workflows.initial_exercises(texts)
-        workflows.initial_cluster_pipeline()
         # Ensure at least one cluster exists for downstream code
-        if not workflows.weaviate_client.get_all_embeddings("ClusterCenter"):
-            workflows.weaviate_client.add_embeddings(
-                "ClusterCenter",
-                [0.1, 0.2, 0.3],  # match your embedding size
-                {"cluster_id": "fake-cluster-id"},
-            )
+        fake_cluster_id = workflows.weaviate_client.get_all_embeddings(CollectionNames.COMPETENCY.value)[3]["properties"]["cluster_id"]
```

Ruff (0.12.2)

Line too long (135 > 88)

(E501)